frontend/src/components/custom-tools/header/Header.jsx

```diff
@@ -85,6 +102,17 @@ function Header({
       });
   };
 
+  const handleConfirmForceExport = useCallback(() => {
+    if (lastExportParams) {
+      const { selectedUsers, toolDetail, isSharedWithEveryone } =
+        lastExportParams;
+
+      handleExport(selectedUsers, toolDetail, isSharedWithEveryone, true);
+    }
+
+    setConfirmModalVisible(false);
```

Comment:
@jagadeeswaran-zipstack In the above code, If the `if` condition fails, we won't display any error message and will silently close the force export confirmation modal. I don't think that would provide a good user experience. Can you display a relevant error message in the notifications component if the condition fails?

Reply:
@tahierhussain In which scenario you think that `if` will condition fail. Even if it fails what error we should throw because its purely state and state being set on catch block and then only we will get to this step.

Reply:
@jagadeeswaran-zipstack In that case, let's remove this `if` statement entirely. I don't think it's necessary. We're first setting the `lastExportParams` state and then opening the confirm modal, so the state will always have a value. I believe it's safe to remove the condition. Also, as a tip for the future, if we add a condition, let's ensure we handle both the success and failure cases properly.

Reply:
Make sense. Yes it's safe to remove that condition.